content/publications/_index.md

```diff
@@ -2,6 +2,9 @@
 bibliography:
 - title: 2020
   items:
+  - yoshitom-vu-2020-sigir
```

Comment:
I do not think this paper should be listed in the lab-wide Publications
page, as it is reserved for the papers done within the lab’s research
line (basically, has Marco as one of the authors).

```suggestion
```